src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCaseEquivalences.Data.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Is this adding a BOM?  The file isn't changing otherwise.

Reply:
I'll check that. There is space difference. I just regenerated this file and decided to keep the newly generated file to ensure it works even if there are no actual changes.

Reply:
Also, how come the file isn't changing? Does that mean that Unicode 15 didn't introduce any new case mappings? I would have expected at least some edits here otherwise.

Reply:
> Does that mean that Unicode 15 didn't introduce any new case mappings?

Right, there is no change in the casing data. You can check all changes in Unicode 15 [here](https://home.unicode.org/announcing-the-unicode-standard-version-15-0/)

> Is this adding a BOM? The file isn't changing otherwise.

I confirmed the change is in the BOM. The newly generated file doesn't include the BOM while the old file had it. I didn't change anything in the tool for how we write to the file, so I expect someone edited this file once inside the editor which brought the BOM. Now the question, do we need to keep the BOM when we generate the file?

Reply:
No need to keep it.  At one point we actually did a pass to remove them all, but they've crept back in.